test/jdk/java/text/Format/NumberFormat/CurrencyFormat.java

```diff
@@ -58,7 +58,7 @@ static void testFormatting() {
             Locale.JAPAN,
             Locale.GERMANY,
             Locale.ITALY,
-            new Locale("it", "IT", "EURO"),
+            Locale.of("it", "IT", "EURO"),
             Locale.forLanguageTag("de-AT"),
             Locale.forLanguageTag("fr-CH"),
```

Comment:
Use the new factory?

Reply:
Ok, I didn't realize the existing method is preferred over the new method in creating a Locale. The javadoc does state that it does not make any syntactic checks. That's good to know.